### ISSUE TITLE
Account v online account

### ISSUE THREAD

**Issue by AngelaMoore** [dwp/dwp-style-guide#64]
We use 'account' and 'online account' inconsistently. At a previous chat about this, we agreed that 'online account' is a useful prompt for people who might not be super-digital.

Should we (for example) use 'account' when someone is in the service and 'online account' when we are telling them to sign in?

**Comment by JohnTurnbull99** [comment]
Agreed - 'online account' when communicating offline eg by SMS (phone scripts?)

Entry in style guide under 'Account'.
